.gitignore

```diff
@@ -55,7 +55,9 @@ docs/_build/
 .env
 .venv
 env/
+pymc4-env/
 venv/
+pymc4-venv/
```

Comment:
```suggestion
```

Reply:
Actually, this was intentional: the Makefile creates these directories: https://github.com/pymc-devs/pymc4/blob/master/Makefile#L28

I almost committed it while making this PR 😄